Makefile

```diff
@@ -271,4 +271,24 @@ dockerized-build-amd64:
 
 # Use vendor instead of proxy when building w/ vendor folder
 dockerized-build-vendor:
-	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
\ No newline at end of file
+	$(DOCKER_BUILD_FROM_SOURCE) --build-arg GO111MODULE=off .
+
+.PHONY: create-eks-cluster
+create-eks-cluster:
+	@eksctl create cluster \
```

Comment:
just curious, but does this choose the default VPC to launch in and any subnets in those zones within that VPC?